### PR TITLE
CEM-2010 Updated 3 interfaces using extends in CustomerController

### PR DESCRIPTION
**Pull request by rmc3408**

### CODE REVIEW

File: src/app/controllers/CustomerController.ts [hunk 1]
```diff
@@ -47,19 +47,16 @@ export interface Customer
     LoyaltyFunction,
     CommonCustomerProperties {}
 
-export interface CustomerOptions {
-  profile_picture?: string;
-  birthday?: string;
+export interface CustomerOptions extends Pick<CommonCustomerProperties, 'profile_picture' | 'birthday'>{
```
Comment (ralph-dev) [inline src/app/controllers/CustomerController.ts:50]: Where is the Partial? Aren't these optional?

**Comment by rmc3408** [comment]
Ok, So can i use `Partial<Pick<CommonCustomerProperties, 'profile_picture' | 'birthday'`.

My understanding is Partial will return T or undefined? is that right?

File: src/app/controllers/CustomerController.ts (hunk 1 above)
Comment (rmc3408) [reply src/app/controllers/CustomerController.ts:50]: Ok, So can i use `Partial<Pick<CommonCustomerProperties, 'profile_picture' | 'birthday'`.

My understanding is Partial will return T or undefined? is that right?

Comment (rmc3408) [reply src/app/controllers/CustomerController.ts]: In the Customer interface is missing 3 keys according with GraphQL schema. ( supercharge_coupon, deliverer_profile and delivery_adresses)

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**